src/core_plugins/kibana/public/management/sections/objects/_view.html

```diff
@@ -1,36 +1,127 @@
-<kbn-management-app section="kibana">
-  <kbn-management-objects-view class="container">
-    <div class="pull-right" style="margin-top: 20px;">
-      <a href="{{ link }}" class="btn btn-primary">View {{ title }}</a>
-      <a confirm-click="delete()" class="btn btn-danger"><i class="fa fa-trash-o"></i> Delete {{ title }} Object</a>
+<kbn-management-app section="kibana" class="kuiView">
+  <kbn-management-objects-view class="kuiViewContent kuiViewContent--constrainedWidth">
+    <!-- Header -->
+    <div class="kuiViewContentItem kuiSubHeader">
+      <h1 class="kuiTitle">
+        Edit {{ title }}
+      </h1>
+
+      <div class="kuiButtonGroup">
+        <a
+          class="kuiButton kuiButton--basic kuiButton--iconText"
+          href="{{ link }}"
+        >
```

Comment:
Haha yeah that was me. I'll add further explanation to the style guide about it. I've found it to be useful for a few reasons...

**Tag alignment**

When I'm scanning markup, it's easier for me to discern the parent child relationship between the enclosing tags and the content when the tags share the same indentation, and the child is indented one level deeper. Conceptually, this makes sense to me too, since the enclosing tags become a parent node and the text content becomes a child node. So the markup mirrors the DOM a little more clearly.

**Bracket alignment**

When there are multiple attributes in the opening tag, having the closing bracket aligned to the same depth as the opening bracket helps me visually match them and group them. For example, I might scan and think `content` is an attribute if the closing bracket is on the same line:

```html
<tag
   attr="val">
  content
</tag>
```

This might only make sense to me, but I also think of this way of indenting brackets as similar to the way you would indent the opening and closing braces and brackets in object and array definitions. They have the same indentation to form a visual alignment, which helps visually enclose their content:

```javascript
const object = {
  'a': {
  },
  'b': [
    0,
    1,
    2,
  ],
};
```

**Separation between tag and attributes**

By putting the attributes on the same line it's easy for me to visually separate them from the tag. This is important because the tag defines the base identity of the element, but the attributes are used to customize it and define its role in the UI.

**Easier attribute comparison**

When there are a number of elements with similar, yet slightly different attributes, having each attribute on its own row makes it easier for me to scan, find the pattern, and discern differences.

```html
<button
  class="kuiButton kuiButton--basic"
  ng-click="edit(service, item)"
  aria-label="Edit"
>
  <span
    aria-hidden="true"
    class="kuiButton__icon kuiIcon fa-pencil"
  ></span>
</button>

<button
  class="kuiButton kuiButton--basic"
  ng-click="open(item)"
  aria-label="View"
>
  <span
    aria-hidden="true"
    class="kuiButton__icon kuiIcon fa-eye"
  ></span>
</button>
```

In this example, I need to scan and check that:

1. Both buttons follow the same general structure: they're both default icon buttons.
2. Both buttons have `aria-label` attributes.
3. The `aria-label` attribute is different for each and has the correct value.
4. Both buttons contain icons.
5. Both icons have `aria-hidden="true"` attributes.
5. Each icon is a different (and correct) kind of icon.

This would be much harder for me to do if they were formatted like this:

```html
<button class="kuiButton kuiButton--basic" ng-click="edit(service, item)" aria-label="Edit">
  <span aria-hidden="true" class="kuiButton__icon kuiIcon fa-pencil"></span>
</button>

<button class="kuiButton kuiButton--basic" ng-click="open(item)" aria-label="View">
  <span aria-hidden="true" class="kuiButton__icon kuiIcon fa-eye"></span>
</button>
```

I think the difficulty of scanning this kind of markup is that you have to scan right, jump to the next line, start at the left side, and continue. It's sort of like the reason we write things in list form -- because lists are easier to scan than long comma-separated sentences.